src/components/countdown/code/views/countdownView.jsx

```diff
@@ -0,0 +1,49 @@
+var React = require('react');
```

Comment:
We need to add a `componentWillUnmount` method to this class, it should call `this.countdownManager.stop();`